configuration/src/main/java/io/airlift/configuration/ConfigurationInspector.java

```diff
@@ -109,7 +109,7 @@ private ConfigRecord(ConfigurationFactory configurationFactory, ConfigurationPro
                     description = "";
                 }
 
-                if (attribute.isHidden()) {
+                if (attribute.isHidden() && !Boolean.getBoolean("config.includeHidden")) {
```

Comment:
This requires `-Dconfig.includeHidden=true`

Reply:
I think that is fine for this.